inspirehep/dojson/hep/model.py

```diff
@@ -53,9 +57,26 @@ def add_book_info(record, blob):
     return record
 
 
+def add_inspire_category_from_arxiv_categories(record, blob):
+    if not record.get('arxiv_eprints') or record.get('inspire_categories'):
+        return record
+
+    record.setdefault('inspire_categories', [])
+    for arxiv_category in record.get('arxiv_eprints').get('categories'):
```

Comment:
It might be a little slower, but the access string is easily greppable, which is going to be very useful when doing codebase-wide modifications (like the one you're doing): you just have to
```
$ af field_categories.term
```
and you get all the uses.

It's also our antidote to JSON Driven Development: http://okigiveup.net/arguments-against-json-driven-development/.